includes/utils.sh

```diff
@@ -256,15 +256,15 @@ then
 		if [ "${RET0}" != "" ]
 		then
 			RET2=`eval norminette $RET0 2>&1`
+			RET4=$?
 			echo "$RET2" > "$RETURNPATH"/.mynorminette
 			RET2=`cat .mynorminette | grep Error`
 			RET3=`cat .mynorminette | grep Warning`
-			RET4=`cat .mynorminette | grep "command not found"`
 			if [ "$RET2" == "" -a "$RET3" == "" -a "$RET4" == "" ]
```

Comment:
´$RET4´  will never be empty if you assign the last exit status, will not?

And are you sure it will take the exit status code of the ´eval´ command and not the assignation ´RET4=...´ ?